runtime/onert/core/CMakeLists.txt

```
@@ -2,7 +2,10 @@ file(GLOB_RECURSE SOURCES "src/*.cc")
 file(GLOB_RECURSE TESTS "*.test.cc")
 list(REMOVE_ITEM SOURCES ${TESTS})
 
-nnfw_find_package(Ruy REQUIRED)
+if(NOT BUILD_MINMAX_H5DUMPER)
+  file(GLOB_RECURSE H5DUMPER_SRC "src/dumper/h5/*.cc")
+  list(REMOVE_ITEM SOURCES ${H5DUMPER_SRC})
+endif(NOT BUILD_MINMAX_H5DUMPER)
```

IMO, we may not need `BUILD_MINMAX_H5DUMPER` cmake option.

```suggestion
nnfw_find_package(HDF5 QUIET)
if(NOT HDF5_FOUND)
  file(GLOB_RECURSE H5DUMPER_SRC "src/dumper/h5/*.cc")
  list(REMOVE_ITEM SOURCES ${H5DUMPER_SRC})
endif(NOT HDF5_FOUND)
```

---

@hseok-oh I did in above way for `onert_run` to dump h5. `onert` silently disable hdf5 related features. But I chose to introduce `BUILD_MINMAX_H5Dumper` since it is in `core`, not `tool`. What do you expect for `minmax-recorder` to do when `HDF5` is not available? Currently `hdf5` format is the only format `minmax-recorder` supports. Do you want to create dummy dumper which does nothing silently when `hdf5` is not available?

---

I see.